packages/synapse-react-client/src/lib/containers/AccessRequirementListV2/RequirementItem/SelfSignAccessRequirementItem.tsx

```diff
@@ -0,0 +1,161 @@
+import React, { useState } from 'react'
+import MarkdownSynapse from '../../markdown/MarkdownSynapse'
+import {
+  AccessApproval,
+  ApprovalState,
+  isTermsOfUseAccessRequirement,
+  SelfSignAccessRequirement,
+  TermsOfUseAccessRequirement,
+} from '../../../utils/synapseTypes'
+import { useSynapseContext } from '../../../utils/SynapseContext'
+import { Box, ButtonProps, Link, Typography } from '@mui/material'
+import {
+  useCreateAccessApproval,
+  useGetAccessRequirementStatus,
+  useGetAccessRequirementWikiPageKey,
+  useGetCurrentUserBundle,
+  useGetCurrentUserProfile,
+} from '../../../utils/hooks/SynapseAPI'
+import RequirementItem from './RequirementItem'
+import { RequirementItemState } from '../AccessApprovalCheckMarkV2'
+
+export type SelfSignAccessRequirementItemProps = {
+  accessRequirement: SelfSignAccessRequirement | TermsOfUseAccessRequirement
+  onHide: () => void
+}
+
+/**
+ * Renders a {@link RequirementItem} for an self-sign Access Requirement, including {@link SelfSignAccessRequirement} or
+ * {@link TermsOfUseAccessRequirement}.
+ *
+ * For {@link ManagedACTAccessRequirement}s, see {@link RequestDataAccess}
+ */
+export default function SelfSignAccessRequirementItem(
+  props: SelfSignAccessRequirementItemProps,
+) {
+  const { accessRequirement, onHide } = props
+  const { accessToken } = useSynapseContext()
+  const isSignedIn = !!accessToken
+  const isTermsOfUse = isTermsOfUseAccessRequirement(accessRequirement)
+
+  const { data: user } = useGetCurrentUserProfile()
+  const { data: userBundle } = useGetCurrentUserBundle()
+
+  const certificationRequirementNotMet =
+    isSignedIn &&
+    userBundle &&
+    !isTermsOfUse &&
+    accessRequirement.isCertifiedUserRequired &&
+    !userBundle.isCertified
+  const verificationRequirementNotMet =
+    isSignedIn &&
+    userBundle &&
+    !isTermsOfUse &&
+    accessRequirement.isValidatedProfileRequired &&
+    !userBundle.isVerified
+
+  const { data: accessRequirementStatus, isLoading: isLoadingStatus } =
+    useGetAccessRequirementStatus(String(accessRequirement.id))
+
+  const isApproved = accessRequirementStatus?.isApproved
+
+  const { data: wikiPage } = useGetAccessRequirementWikiPageKey(
+    accessRequirement.id.toString(),
+    // TermsOfUse ARs have the terms embedded in the AR, there is no Wiki.
```

Comment:
Would love more context on why we have ToU ARs? Are these basically just legacy SelfSignedARs?